tools/rpki-rov.1

```diff
@@ -0,0 +1,28 @@
+.\"
+.\" This file is part of RTRlib.
+.\"
+.\" This file is subject to the terms and conditions of the MIT license.
+.\" See the file LICENSE in the top level directory for more details.
+.\"
+.\" Website: http://rtrlib.realmv6.org/
+.\"
+.TH "rpki-rov" "1"
+.SH NAME
+rpki-rov - rpki route origin validator
+.SH SYNOPSIS
+.B rpki-rov
+[\fIhost\fR] [\fIport\fR]
+.SH DESCRIPTION
+\fBrpki-rov\fR is a cli interface for rpki route origin validation.
```

Comment:
s/cli interface for rpki route origin validation/command line interface that implemends RPKI route origin validation/